.github/workflows/ci-linting.yml

```diff
@@ -1,15 +1,16 @@
 name: "Linting"
 
 on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
-  release:
-    types:
-      - created
+  workflow_dispatch:
+  # push:
+  #   branches:
+  #     - main
+  # pull_request:
+  #   branches:
+  #     - main
+  # release:
+  #   types:
+  #     - created
```

Comment:
Really want to push commented lines?

Reply:
apologies @rrw-zilliqa, I raised this pr for test, purposes. I just closing this